docs/configure/babel.md

```diff
@@ -18,9 +18,7 @@ Check out our [source](https://github.com/storybookjs/storybook/blob/next/lib/co
 
 ### Custom config file
 
-If your project has a `.babelrc` file, we'll use that instead of the default config file.
-
-You can also place a `.storybook/.babelrc` file to use a special configuration for Storybook only.
+If your project has a `.storybook/.babelrc` file, we'll use that instead of the default config file.
```

Comment:
According to the code https://github.com/storybookjs/storybook/blob/next/lib/core-common/src/utils/load-custom-babel-config.ts#L79 there are more filenames considered for loading in the `.storybook` folder

- .babelrc
- .babelrc.json
- .babelrc.js
- babel.config.json
- babel.config.js

Reply:
Important to note as well that the recommended setting for Babel 7 is to use `babel.config.json`, lots of people are renaming their configuration file so it would be nice indeed to see that extension in the docs

Ref: https://babeljs.io/docs/en/config-files#6x-vs-7x-babelrc-loading